src/Features/Core/Portable/UnusedReferences/UnusedReferencesRemover.cs

```diff
@@ -236,25 +269,27 @@ internal static ImmutableArray<string> GetAllCompilationAssemblies(ReferenceInfo
                 .ToImmutableArray();
         }
 
-        public static async Task<Solution> UpdateReferencesAsync(
+        public static async Task<(bool Success, Solution UpdatedSolution)> UpdateReferencesAsync(
```

Comment:
do we need the success value?  coudl we use 'null' to indicate failure?  in other words, is `(false, not-null)` ever something we use and need?

Reply:
Going to move this change to a branch where I can add strings and report incomplete results to the user.